File: module-load-balance-gateway/src/main/java/com/xmmufan/permission/repository/UserGroupRepository.java
package com.xmmufan.permission.repository;

import com.xmmufan.permission.domain.rbac.UserGroup;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

/**
 * @author 黄源钦
 * @Description
 * @Date 13:41
 */
@Repository
public interface UserGroupRepository extends JpaRepository<UserGroup,String> {
}

File: module-load-balance-gateway/src/main/java/com/xmmufan/permission/repository/UserGroupRepository.kt
package com.xmmufan.permission.repository

import com.xmmufan.permission.domain.rbac.UserGroup
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

/**
 * @author 黄源钦
 * @Description
 * @Date 13:41
 */
@Repository
interface UserGroupRepository : JpaRepository<UserGroup, String>
